types/react-css-collapse/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for react-css-collapse 3.6
+// Project: https://github.com/SparebankenVest/react-css-collapse
+// Definitions by: Daniel Ford <https://github.com/dford07>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+declare module 'react-css-collapse' {
```

Comment:
I think this module declaration is redundant - the exports already come from `react-css-collapse`.

Reply:
is it worth me changing this?